boards/clue_nrf52840/layout.ld

```diff
@@ -0,0 +1,12 @@
+MEMORY
+{
+  # Make space for the UF2 bootloader
+  rom (rx)  : ORIGIN = 0x00026000, LENGTH = 192K
+  prog (rx) : ORIGIN = 0x00056000, LENGTH = 640K
```

Comment:
Wow the bootloader is that large??

Reply:
Yes, it has the UF2 bootloader that provides a USB drive flash capability.